test/lambda/runtime-promises.test.js

```diff
@@ -26,7 +26,7 @@ test.serial.before(async () => {
     handler: 'runtime_promises.handler',
     // Using Node 6.10 gives a more thorough test since this isn't normally
```

Comment:
Should this comment be removed?

Reply:
I just did a find/replace.  Didn't see this one without any spaces.